indexer/src/block_listener.rs

```diff
@@ -355,6 +355,7 @@ mod tests {
                 nonce: 0,
                 receiver_id: da_contract_id,
                 actions: vec![ActionView::CreateAccount],
+                priority_fee: 0,
```

Comment:
was this breaking? i'm quite shocked by this since it should be ignorable. Good to know!